### ISSUE TITLE
Cannot find module 'typescript' error with Node v12.22 after installing @nrwl/angular 

### ISSUE THREAD

**Issue by abf7d** [nrwl/nx#12943]
## Current Behavior
I am trying to create a react app workspace and add an angular app with Nx v13.2. I need to use this version because my version of Node is locked into v12.22 right now and therefore need to use Angular 13 and React 17. When I create a react workspace, I am able to run it with no probelms. I then install @nrwl/angular@13.2 and run `nx g @nrwl/angular:app angular-demo`. I get the error: `Cannot find module 'typescript`. When I run npm install which prints info about missing peer dependencies, I see the following statement:
```
npm WARN notsup Unsupported engine for quadbin@0.1.5: wanted: {"node":">=14"} (current: {"node":"12.22.12","npm":"6.14.16"})
npm WARN notsup Not compatible with your version of node/npm: quadbin@0.1.5
```
I'm fairly certain that the problem is occurring because the version of Node I have is older.

## Expected Behavior
When I download @nrwl/angular@13.2 I expect that I should be able to create an angular app while running Node v12.22

## Steps to Reproduce
1. set environment to run Node v12.22
2. run npx create-nx-workspace@13.2 and select the `single react app` option
3. run `npm i @nrwl/angular@13.2` to install the angular installer
4. run `nx g @nrwl/angular:app angular-demo` to create an angual app
5. observe the `Cannot find module 'typescript` error

### Failure Logs
Full error that occurs when you try to create an angular project:
```
 >  NX   Cannot find module 'typescript'

   Require stack:
   - C:\Work\sandbox\poc\node_modules\nx\src\project-graph\build-dependencies\typescript-import-locator.js
   - C:\Work\sandbox\poc\node_modules\nx\src\project-graph\build-dependencies\explicit-project-dependencies.js
   - C:\Work\sandbox\poc\node_modules\nx\src\project-graph\build-dependencies\index.js
   - C:\Work\sandbox\poc\node_modules\nx\src\project-graph\build-project-graph.js
   - C:\Work\sandbox\poc\node_modules\nx\src\daemon\server\project-graph-incremental-recomputation.js
   - C:\Work\sandbox\poc\node_modules\nx\src\daemon\server\server.js
   - C:\Work\sandbox\poc\node_modules\nx\src\daemon\server\start.js
```

### Environment

Running `nx report` produces the same typescript error:
```
 >  NX   Cannot find module 'typescript'

   Require stack:
   - C:\Work\sandbox\poc\node_modules\nx\src\project-graph\build-dependencies\typescript-import-locator.js
   - C:\Work\sandbox\poc\node_modules\nx\src\project-graph\build-dependencies\explicit-project-dependencies.js
   - C:\Work\sandbox\poc\node_modules\nx\src\project-graph\build-dependencies\index.js
   - C:\Work\sandbox\poc\node_modules\nx\src\project-graph\build-project-graph.js
   - C:\Work\sandbox\poc\node_modules\nx\src\daemon\server\project-graph-incremental-recomputation.js
   - C:\Work\sandbox\poc\node_modules\nx\src\daemon\server\server.js
   - C:\Work\sandbox\poc\node_modules\nx\src\daemon\server\start.js
  ```

**Comment by FrozenPandaz** [comment]
Sorry, I can't reproduce this error on `Node 12` and `Nx 13.2`.

Can you please retry with the latest version of Nx please? (Nx 15)

**Comment by abf7d** [comment]
I can't use Nx 15. I have to use Nx 13.2 because it uses Angular 13 and React 17. My Node is fixed to 12.22 and neither Angular 14 nor React 18 support Node 12.22.
